docs/user-guides/configuration.md

```diff
@@ -445,7 +452,7 @@ A TLS config allows configuring TLS connections.
   dashboard: <string>
 ```
 
-### Provisioning specification
+#### Provisioning specification
```

Comment:
it's not part of Frontend config so 3 hashes were fine
\+ maybe put this section before Frontend config since it comes before it in the overall config

Reply:
done